internet-indicator@sangorys/files/internet-indicator@sangorys/README.md

```diff
@@ -0,0 +1,19 @@
+Cinnamenu

```

Comment:
This is Cinnamenu's README

Reply:
First of all : thanks for all your valuable comments !!!

My bad. This file should even not exist in this folder. The correct README.md is the one in the root internet-indicator@sangorys folder.
I removed the wrong README.md